src/jest.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
   overrides: [
     {
       env: {
-        es6: true,
+        es2022: true,
```

Comment:
Good point, just a question, is this the same as naming it `esnext`?

Reply:
Yes, the last release was es2021.
Now, es2022 = esnext.
But, ESLint doesn't support setting the [environment](https://eslint.org/docs/user-guide/configuring/language-options#specifying-environments) to `esnext` unfortunately.